### PR TITLE
added rplidar sensor to raw3-3 urdf and bringup

### PR DESCRIPTION
**Pull request by fmirus**

### CODE REVIEW

File: cob_bringup/robots/raw3-3.xml [hunk 1]
```diff
@@ -63,6 +63,9 @@
 		<include file="$(find cob_bringup)/tools/base_collision_observer.launch" >
 			<arg name="robot" value="$(arg robot)" />
 		</include>
+
+		<include file="$(find rplidar_ros)/launch/rplidar.launch" />
```
Comment (fmessmer) [inline cob_bringup/robots/raw3-3.xml]: Is `rplidar_ros` a publically available ROS-package?
If so, you can add a run_dependency to the package.xml.

Comment (fmirus) [inline cob_bringup/robots/raw3-3.xml]: @ipa-fxm yes, the package can be found here: https://github.com/robopeak/rplidar_ros
I have also added the run dependency as suggested.

Comment (fmessmer) [inline cob_bringup/robots/raw3-3.xml]: Thx, but `rplidar_ros` is not released, right?

Comment (mgruhler) [inline cob_bringup/robots/raw3-3.xml]: Has been released into hydro, but not into indigo

**Comment by floweisshardt** [comment]
if this is not released, which means "apt-get installable", we should be extremly carefull with adding this to cob_robots launch files as it will fail launching for normal users (without overlays).

My suggestion if you like to have this in:
- add a launch files argument , e.g. "include_non_released_packages" that will default to false
- put the rplidar.launch into a group which using "if" and "unless" features

**Comment by mgruhler** [comment]
@ipa-frm Done?
@ipa-fmw FYI

**Comment by fmirus** [comment]
@ipa-mig think so :wink:

**Comment by mgruhler** [comment]
:ok:

File: cob_bringup/robots/raw3-3.xml [hunk 2]
```diff
@@ -3,6 +3,7 @@
 
 	<!-- args -->
 	<arg name="robot" value="raw3-3"/>
+	<arg name="include_non_released_packages" default="false"/>
```
Comment (fmessmer) [inline cob_bringup/robots/raw3-3.xml]: Can we use a more related arg name, e.g. `<arg name="use_rplidar" default="false"/>`?

File: cob_bringup/robots/raw3-3.xml [hunk 3]
```diff
@@ -62,7 +63,12 @@
 		</include>
 		<include file="$(find cob_bringup)/tools/base_collision_observer.launch" >
 			<arg name="robot" value="$(arg robot)" />
-		</include>
+    </include>
+
+    <group if="$(arg include_non_released_packages)">
+      <include file="$(find rplidar_ros)/launch/rplidar.launch" />
+    </group>
+
```
Comment (fmessmer) [inline cob_bringup/robots/raw3-3.xml:71]: tabs vs. spaces!

**Comment by fmessmer** [comment]
I'll give this one a "good-to-go" if my two comments are considered....:wink:

File: cob_bringup/robots/raw3-3.xml (hunk 2 above)
Comment (mgruhler) [inline cob_bringup/robots/raw3-3.xml]: This has been done according to @ipa-fmw's comment.
So, what should we do? The current would be a more general approach, the other a more clear one.
I don't mind ;-)

Comment (floweisshardt) [inline cob_bringup/robots/raw3-3.xml]: +1 for version from @ipa-fxm

**Comment by mgruhler** [comment]
@ipa-frm push

**Comment by fmirus** [comment]
@ipa-mig update

**Comment by fmessmer** [comment]
LGTM

**Comment by floweisshardt** [comment]
LGTM

**Pull request merged**